Source/ablastr/constant.H

```diff
@@ -17,13 +17,24 @@ namespace ablastr::constant
     /** Mathematical constants */
     namespace math
     {
-        using namespace amrex::literals;
+        namespace variable_templates
```

Comment:
Since this is for constants, perhaps call it `constant_templates`?

Reply:
Sorry for not having replied earlier, @dpgrote !
Would `const_templates` work according to you?

Reply:
Thanks, it's better, but `const` has a specific meaning in C++ so that could cause confusion.

Reply:
I think that you are right, @dpgrote . I've changed the name to `constant_templates`, as you originally suggested.